translations/ja.json

```
@@ -250,53 +250,53 @@
+	"Failed to save config {{.profile}}": "設定 {{.profile}} の保存に失敗しました",
+	"Failed to save dir": "ディレクトリーの保存に失敗しました",
+	"Failed to save image": "イメージの保存に失敗しました",
+	"Failed to save stdin": "標準出力の保存に失敗しました",
+	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}`.": "NO_PROXY 環境変数の設定に失敗しました。`export NO_PROXY=$NO_PROXY,{{.ip}}` を使用してください。",
+	"Failed to setup certs": "証明書セットアップに失敗しました",
+	"Failed to start container runtime": "コンテナーランタイムの起動に失敗しました",
+	"Failed to start {{.driver}} {{.driver_type}}. Running \"{{.cmd}}\" may fix it: {{.error}}": "{{.driver}} {{.driver_type}} の開始に失敗しました。「{{.cmd}}」実行で解決するかも知れません: {{.error}}",
+	"Failed to stop node {{.name}}": "{{.name}} ノードの停止に失敗しました",
+	"Failed to tag images": "イメージのタグ付与に失敗しました",
+	"Failed to update cluster": "クラスター更新に失敗しました",
+	"Failed to update config": "設定更新に失敗しました",
+	"Failed unmount: {{.error}}": "アンマウントに失敗しました: {{.error}}",
+	"Filter to use only VM Drivers": "VM ドライバーのみ使用するためのフィルタ",
 	"Flags": "フラグ",
-	"Follow": "たどる",
-	"For best results, install kubectl: https://kubernetes.io/docs/tasks/tools/install-kubectl/": "最適な結果を得るには、kubectl を次のサイト https://kubernetes.io/docs/tasks/tools/install-kubectl/ からインストールしてください",
-	"For improved {{.driver}} performance, {{.fix}}": "",
-	"For more information see: https://minikube.sigs.k8s.io/docs/drivers/{{.driver}}": "",
-	"For more information, see: https://minikube.sigs.k8s.io/docs/reference/drivers/none/": "",
-	"For more information, see: {{.url}}": "",
-	"Force environment to be configured for a specified shell: [fish, cmd, powershell, tcsh, bash, zsh], default is auto-detect": "",
-	"Force minikube to perform possibly dangerous operations": "minikube で危険な可能性のある操作を強制的に実行します",
-	"Format output. One of: short|table|json|yaml": "",
-	"Format to print stdout in. Options include: [text,json]": "",
-	"Found docker, but the docker service isn't running. Try restarting the docker service.": "",
-	"Found driver(s) but none were healthy. See above for suggestions how to fix installed drivers.": "",
+	"Follow": "ログフォロー",
+	"For best results, install kubectl: https://kubernetes.io/docs/tasks/tools/install-kubectl/": "最善の結果を得るために、https://kubernetes.io/docs/tasks/tools/install-kubectl/ から kubectl をインストールしてください",
+	"For improved {{.driver}} performance, {{.fix}}": "{{.driver}} の性能向上のため、{{.fix}}",
+	"For more information see: https://minikube.sigs.k8s.io/docs/drivers/{{.driver}}": "追加の詳細情報はこちらを参照してください: https://minikube.sigs.k8s.io/docs/drivers/{{.driver}}",
+	"For more information, see: https://minikube.sigs.k8s.io/docs/reference/drivers/none/": "追加の詳細情報はこちらを参照してください: https://minikube.sigs.k8s.io/docs/reference/drivers/none/",
+	"For more information, see: {{.url}}": "追加の詳細情報はこちらを参照してください: {{.url}}",
+	"Force environment to be configured for a specified shell: [fish, cmd, powershell, tcsh, bash, zsh], default is auto-detect": "特定シェル用の環境設定を行います: [fish, cmd, powershell, tcsh, bash, zsh] (デフォルトは auto-detect)",
```

```suggestion
	"Force environment to be configured for a specified shell: [fish, cmd, powershell, tcsh, bash, zsh], default is auto-detect": "指定されたシェル用の環境設定を強制的に行います: [fish, cmd, powershell, tcsh, bash, zsh] (デフォルトは auto-detect)",
```
This text is a description for `--shell` option. (e.g. `minikube docker-env --shell powershell`)
I think "指定されたシェル" is better because user specifies the shell in this option.
And, "Force" exists in the original text, please add "強制的に".